src/core/types/Query.ts

```diff
@@ -166,6 +173,15 @@ export interface BaseEscrowDepositFragResponse {
   enabled: boolean;
   amount: string;
   token: TokenFragRepsonse;
+  events?: EscrowEventFragResponse[];
+}
+
+export interface EscrowEventFragResponse {
+  __typename: string;
+  timestamp: number;
+  // only on add/remove collateral
+  amount?: string;
```

Comment:
Why did u switch value/amount from string -> number in LineEventFragResponse but add them here as strings? 

They shouldn't be numbers because Javascript literally doesnt support numbers that big so we would get errors on API responses if they were numbers not strings (i think).

Reply:
If I leave `amount` and `value` as `string` type, I receive a typeerror because the `CreditEvent` and `EventWithValue` interfaces in `types/CreditLine.ts` have those fields set to `number` type. The `formatLineWithEvents` function makes a comparison between `CreditEvent` and `LineEventFragResponse` for those properties. 

Should all numeric fields that are `amount` or `value` in `CreditLine.ts` be strings instead of numbers?

Reply:
numeric values in the `CreditLine.ts` and `Query.ts` files for subgraph response, with some exceptions (e.g. `timestamp`, `token.decimals`), are now set to string. Basically, anything that could potentially be a `BigNumber` like `amount` or `value` have been set to string type.

Reply:
ya they should all be strings